### ISSUE TITLE
Error: Component must be a React component

### ISSUE THREAD

**Issue by doctyper** [sheepsteak/react-perf-component#4]
Hello,

I'm seeing this error using `react-perf-component@1.0.1`:

```
Error: Component must be a React component
```

No further details are available. I think 1.0.1 should have been released as 2.0.0 due to the React 15 bump.

**Comment by sheepsteak** [comment]
Are you passing in a stateless component?

When I first made this module they didn't exist so I think they're failing the check to make sure a component is passed in as they're obviously just functions.

Would you mind installing the `stateless` branch and seeing if that works for you?

**Comment by sheepsteak** [comment]
I tried the above and got stateless components working. However, then I found out that `getLastMeasurements` has changed and [it is no longer recommended to use it](https://facebook.github.io/react/blog/2016/04/07/react-v15.html#reactperf.getlastmeasurements-is-opaque):

> The internal performance measurement format is subject to change so, for the time being, we consider the return value of ReactPerf.getLastMeasurements() an opaque data structure that should not be relied upon.

I was using this to work out the average render times so I've had to rip that out and I'll have to find another way to do that in the future. Given that change I agree it should be 2.0.0 (I did this with https://github.com/sheepsteak/react-shallow-testutils but thought I could get away with it here) and I've pushed it up to npm.

Sorry for any inconvenience.
